src/Raven.Server/NotificationCenter/Handlers/ServerNotificationCenterHandler.cs

```diff
@@ -113,23 +112,25 @@ public abstract class ServerNotificationHandlerBase : RequestHandler
                 case RavenServer.AuthenticationStatus.NotYetValid:
                     if (Server.Configuration.Security.AuthenticationEnabled == false)
                         return null;
-                    return s => false; // deny everything
+                    return (_, __) => false; // deny everything
 
                 case RavenServer.AuthenticationStatus.Operator:
                 case RavenServer.AuthenticationStatus.ClusterAdmin:
                     return null;
 
                 case RavenServer.AuthenticationStatus.Allowed:
-                    return database =>
+                    return (database, requireWrite) =>
```

Comment:
I don't understand under what scenario `requireWrite` would be true?

Reply:
Dismiss and Postpone